fpga_interchange/testarch_generators/generate_testarch.py

```diff
@@ -186,17 +209,23 @@ def make_tile_type(self, tile_type_name, site_types):
         # TODO: const. wires
 
     def make_device_grid(self):
-        
+
         for y in range(self.grid_size[1]):
             for x in range(self.grid_size[0]):
 
+                is_0_0 = x ==0 and y == 0
                 is_perimeter = y in [0, self.grid_size[1] - 1] or \
                                x in [0, self.grid_size[0] - 1]
+                is_centre = y == self.grid_size[1] // 2 and x == self.grid_size[0] // 2
```

Comment:
Nextpnr_emit expects tile 0,0 to be dummy/null tile.
Also nextpnr_emmit needs some why to inject GND and VCC signals, this is now done by PWR tile

Reply:
Yeah, this is fair for now - eventually we do want to find a better solution to that